script/utils/transferOwnership.sh

```diff
@@ -1,142 +1,193 @@
 #!/bin/bash
 
-# Default to dry run mode
+# ANSI escape codes for text color
+RED="\033[31m"
+GREEN="\033[32m"
+YELLOW="\033[33m"
+BOLD="\033[1m"
+RESET="\033[0m"
+
+usage() {
+    echo "Usage: $0 --rpc-url <uri> --owner-address <address> --upgrader-address <address> --eoracle-chain-id <id> --private-key <private key> [--dry-run]"
+    exit 1
+}
+
 dry_run=true
 
-# Check if required environment variables are set
+# Parse named arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --rpc-url)
+            TARGET_RPC_URL=${2}
+            shift 2
+            ;;
+        --owner-address)
+            OWNER_ADDRESS=${2}
+            shift 2
+            ;;
+        --upgrader-address)
+            UPGRADER_ADDRESS=${2}
+            shift 2
+            ;;
+        --eoracle-chain-id)
+            EORACLE_CHAIN_ID=${2}
+            shift 2
+            ;;
```

Comment:
change to `chain` (values `mainnet`, `testnet`, `devnet`, `local`)
you can have constants for the chains